src/docs/release/breaking-changes/android-v1-embedding-create-deprecation.md

```diff
@@ -57,9 +63,11 @@ encountering this decreased as plugin developers create and publish v2 plugins.
 
 ## Migration guide
 
-See https://flutter.dev/go/android-project-migration.
+See the [migration guide][].
+
+[migration guide]: /go/android-project-migration
 
 ## Timeline
 
 Landed in version: 1.20.0-8.0<br>
-In stable release: not yet
+In stable release: 1.20
```

Comment:
Looking at https://github.com/flutter/flutter/commit/8bd2e6585b0eda5ccdc10c1e1d35105e11f84424, it seems like it was first brought into the 1.21 stable branch.

Reply:
In other words, it's not yet on stable, since latest stable is 1.20. I'll fix it.

Reply:
And thanks for that correction!